Modules/Video/IO/wrapping/itkVideoFileReader.wrap

```diff
@@ -0,0 +1,12 @@
+itk_wrap_include("itkImage.h")
+itk_wrap_include("itkVideoStream.h")
+
+UNIQUE(image_types "UC;ULL;D;${WRAP_ITK_ALL_TYPES}")
+itk_wrap_class("itk::VideoFileReader" POINTER)
+  foreach(t ${image_types})
```

Comment:
@dzenanz Is this style what you mean?

Reply:
I thought  that two classes are wrapped here, `VideoFileReader` and `VideoStream`. So I meant:
```cmake

# wrap first class
itk_wrap_class("itk::VideoFileReader" POINTER)

# now wrap second class
UNIQUE(image_types "UC;ULL;D;${WRAP_ITK_ALL_TYPES}")
foreach(t ${image_types})
  # indentation comes inside a for loop
  ...
  foreach(...)
    # another indentation level
    ...
    # itk_wrap_template for itk::VideoStream
  endforeach()
endforeach()
```
now I see that `VideoFileReader` is wrapped for template parameter `VideoStream`, so my suggestion can be disregarded.

Reply:
Perfect, no problem!